### PR TITLE
Print the console URL upon successful cluster deploy.

### PR DESCRIPTION
**Pull request by kwoodson**
Printing the console output upon successful cluster deploy.

fixes https://github.com/Azure/acs-engine/issues/2891
fixes https://github.com/Azure/acs-engine/issues/2554

@jim-minter

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -366,5 +366,10 @@ func (dc *deployCmd) run() error {
 		log.Fatalln(err)
 	}
 
+	if dc.containerService.Properties.OrchestratorProfile.OrchestratorType == api.OpenShift {
+		// TODO: when the Azure client library is updated, read this from the template `masterFQDN` output
+		fmt.Printf("OpenShift web UI avaialble at https://%s.%s.cloudapp.azure.com:8443/\n", dc.containerService.Properties.MasterProfile.DNSPrefix, dc.location)
```
Comment (kargakis) [inline cmd/deploy.go]: typo: avaialble

Comment (kwoodson) [reply cmd/deploy.go]: Fixed.  Thanks!

**Comment by kwoodson** [comment]
@CecileRobertMichon, Would you mind taking a look at this one?  Thanks in advance.

**Pull request merged**